app/grandchallenge/reader_studies/views.py

```diff
@@ -772,15 +795,33 @@ def get_success_url(self):
         )
 
 
-class AnswersRemoveGroundTruth(AnswerBatchDelete):
-    success_message = "Ground Truth deleted"
+class ReaderStudyGroundTruthDelete(AnswerBatchDelete):
+    template_name = "reader_studies/ground_truth_confirm_delete.html"
+    success_message = "Ground truth successfully deleted"
+
+    def get_success_url(self):
+        return reverse(
+            "reader-studies:ground-truth",
+            kwargs={"slug": self.reader_study.slug},
+        )
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["object"] = self.reader_study
+        return context
 
     def get_queryset(self):
         return Answer.objects.filter(
             question__reader_study=self.reader_study,
             is_ground_truth=True,
         )
 
+    def form_valid(self, *args, **kwargs):
+        Answer.objects.filter(question__reader_study=self.reader_study).update(
```

Comment:
No need to re-calculate scores here: this view will explicilty remove all ground truth so all scores are null.